spec/requests/app_spec.rb

```diff
@@ -46,6 +46,18 @@
     end
   end
 
+  describe 'POST /prayer_list' do
+    let(:pdf_file) { Rack::Test::UploadedFile.new('spec/fixtures/sample.pdf', 'application/pdf') }
```

Comment:
This is the downside of `git add -p`, you can sometimes miss new files.

Reply:
Oh yikes did I miss the PDF file? whoops

Reply:
It doesn't show it here as a new file and is greyed out in the file system. When I click on it in VScode it's jibberish. Could this be because of encoding maybe?

Reply:
Yeah PDFs are encoded in a strange way. They have a certain header at the beginning that tells the thing reading it that it's a PDF, and usually you CAN open it in VSCode but it won't be fully readable. Treat it like you'd treat an image that you're committing to source control.